ui/internal-utilities/src/utils/time-series/mockTimeSeriesResponse.ts

```diff
@@ -11,10 +11,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { PrometheusDatasource } from '@perses-dev/prometheus-plugin';
+import { MatrixData } from '@perses-dev/prometheus-plugin';
 
-type PrometheusClient = ReturnType<(typeof PrometheusDatasource)['createClient']>;
-type RangeQuery = Awaited<ReturnType<PrometheusClient['rangeQuery']>>;
+type MockResponse = {
+  status: string;
+  data: Record<string, unknown>;
+};
```

Comment:
I simplified the typing here because I was having a really hard time getting all the types to play nicely with msw and we don't need quite this level of typing for the mocks. We still get the basics from using `MatrixData` for the results and it removes the need to infer those extra types that aren't exported.